### PR TITLE
APP-235: [Java] Basic level Apps: Events

### PR DESCRIPTION
**Pull request by Rib47**

### CODE REVIEW

File: eventdemo/resources/chatEventClassFamily.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+[
```
Comment (Dubland) [inline eventdemo/resources/chatEventClassFamily.json:1]: Why u add chatEventClassFamily.json? We have chat_event_class_event.avsc and chat_event_class_message.avsc in eventdemo/resources folder

**Review by Slash32 — CHANGES_REQUESTED** [review]
The comments must be addressed before merge.

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

File: eventdemo/resources/chatEventClassFamily.json (hunk 1 above)
Comment (Rib47) [reply eventdemo/resources/chatEventClassFamily.json:1]: You would better ask "Why you don't delete these **chat_event_class_event.avsc** and **chat_event_class_message.avsc** files from **eventdemo/resources** folder?" 
They are not used in **EventDemoBuilder**.

Fixed.

**Comment by Slash32** [comment]
@kaa-leeroy test this hard please

**Comment by Slash32** [comment]
@kaa-leeroy test this hard please

**Comment by Slash32** [comment]
@kaa-leeroy test this hard please

**Comment by Slash32** [comment]
@kaa-leeroy test this hard please

**Comment by Slash32** [comment]
@kaa-leeroy test this hard please

**Comment by Slash32** [comment]
@kaa-leeroy test this hard please

**Comment by Slash32** [comment]
@kaa-leeroy test this hard please

**Comment by vadimol** [comment]
@kaa-leeroy test this hard please

**Review by Slash32 — APPROVED** [review]

**Pull request merged**